### PR TITLE
Some simplifications

### PR DESCRIPTION
**Pull request by noplanman**
A bunch of code simplifications and fixes.

### CODE REVIEW

File: showstats.php [hunk 1]
```diff
@@ -17,18 +17,15 @@
 $apikey = pg_fetch_all($result);
 $upti   = curl_init();
 $key    = $apikey[0]['stats_apikey'];
-$data   = ['all_time_uptime_ratio' => 1, 'format' => 'json', 'custom_uptime_ratios' => '7-30-60-90', 'response_times' => 1, 'response_times_average' => 86400, 'api_key' => $key, 'callback' => 'jsonpUptimeRobot'];
+$data   = ['all_time_uptime_ratio' => 1, 'format' => 'json', 'custom_uptime_ratios' => '7-30-60-90', 'response_times' => 1, 'response_times_average' => 86400, 'api_key' => $key];
 curl_setopt($upti, CURLOPT_URL, 'https://api.uptimerobot.com/v2/getMonitors');
```
Comment (noplanman) [inline showstats.php]: Any reason why you're using v2 here? Found nothing about it in the docs.

**Comment by dmorley** [comment]
Bleeding edge! I don't remember why I started to move to v2, maybe off an 
email they sent, they have v2 docs on their site

**Comment by noplanman** [comment]
Do you have a link?
All I find is [this page](https://uptimerobot.com/api), which doesn't mention v2 anywhere.

Is it maybe in the members area when logged into an account?

**Comment by dmorley** [comment]
https://uptimerobot.com/apiv2

**Comment by noplanman** [comment]
Ah, it's in beta, that's why there's no direct link from the main website.
Do you want to change all requests to use beta or wait till it's stable?

I'm all for bleeding edge, but beta still is beta 😕

**Comment by dmorley** [comment]
I think that's why I left the pull.php on the old one, this one is just a 
rare manual hit.

**Pull request merged**